docs/Getting Started/OpenSUSE/Opensuse Leap Root on ZFS.rst

```diff
@@ -0,0 +1,1057 @@
+    ENV{DM_NAME}!="", SYMLINK+="dm-name-$env{DM_NAME}"' >> /etc/udev/rules.d/99-local-crypt.rules
+
+
+#. Install GRUB
+
+   Choose one of the following options:
+
+   - Install GRUB for UEFI booting::
+
+        zypper in grub2 dosfstools os-prober
+        mkdosfs -F 32 -s 1 -n EFI ${DISK}-part2
+        mkdir /boot/efi
+        echo PARTUUID=$(blkid -s PARTUUID -o value ${DISK}-part2) \
+           /boot/efi vfat nofail,x-systemd.device-timeout=1 0 1 >> /etc/fstab
+        mount /boot/efi
+
+     **Notes:**
+
+     - The ``-s 1`` for ``mkdosfs`` is only necessary for drives which present
+        4 KiB logical sectors (“4Kn” drives) to meet the minimum cluster size
+        (given the partition size of 512 MiB) for FAT32. It also works fine on
+        drives which present 512 B sectors.
+     - For a mirror or raidz topology, this step only installs GRUB on the
+       first disk. The other disk(s) will be handled later.
+
+#. Optional: Remove os-prober::
+
+     zypper remove os-prober
+
+   This avoids error messages from `update-bootloader`.  `os-prober` is only
+   necessary in dual-boot configurations.
+
+#. Set a root password::
+
+     passwd
+
+#. Enable importing bpool
+
+   This ensures that ``bpool`` is always imported, regardless of whether
+   ``/etc/zfs/zpool.cache`` exists, whether it is in the cachefile or not,
+   or whether ``zfs-import-scan.service`` is enabled.
+
+   ::
+
+         vi /etc/systemd/system/zfs-import-bpool.service
+
+   .. code-block:: ini
+
+         [Unit]
+         DefaultDependencies=no
+         Before=zfs-import-scan.service
+         Before=zfs-import-cache.service
+
+         [Service]
+         Type=oneshot
+         RemainAfterExit=yes
+         ExecStart=/sbin/zpool import -N -o cachefile=none bpool
+
+         [Install]
+         WantedBy=zfs-import.target
+
+   ::
+
+     systemctl enable zfs-import-bpool.service
+
+#. Optional (but recommended): Mount a tmpfs to ``/tmp``
+
+   If you chose to create a ``/tmp`` dataset above, skip this step, as they
+   are mutually exclusive choices. Otherwise, you can put ``/tmp`` on a
+   tmpfs (RAM filesystem) by enabling the ``tmp.mount`` unit.
+
+   ::
+
+     cp /usr/share/systemd/tmp.mount /etc/systemd/system/
+     systemctl enable tmp.mount
+
+Step 6: GRUB Installation
+-------------------------
+
+
+#. Add zfs module into dracut::
+
+     echo 'zfs'>> /etc/modules-load.d/zfs.conf
+
+
+#. Verify that the ZFS boot filesystem is recognized::
+
+     grub2-probe /boot
+
+   Output must be `zfs`
+
+#. If you having trouble with `grub2-probe` command make this::
+
+     echo 'export ZPOOL_VDEV_NAME_PATH=YES' >> /etc/profile
+     export ZPOOL_VDEV_NAME_PATH=YES
+
+   then go back to `grub2-probe` step.
+
+#. Refresh kernel files::
+
+    kernel-install add $(uname -r) /boot/vmlinuz-$(uname -r)
+
+#. Refresh the initrd files::
+
+     mkinitrd
+
+   **Note:** When using LUKS, this will print “WARNING could not determine
```

Comment:
This still references an Ubuntu bug report, which is clearly not applicable to openSUSE. At a minimum, that should be updated. But this brings up a more fundamental question: Have you tested the LUKS stuff in openSUSE? If not, you should either do that or rip out all of the LUKS stuff. Don't feel obligated to support LUKS scenarios if you don't want to. This is volunteer work. :)